third_party/istio-1.1-latest

```diff
@@ -1 +1 @@
-istio-1.1.3
\ No newline at end of file
```

Comment:
We cannot add an empty line at the end, since it is a symbolic link instead of a file.
If we force to add an empty line, the link will be broken. @mattmoor-sockpuppet

Reply:
Yeah, just ignore this.  The bot has a dumb heuristic right now for avoiding symlinks, which doesn't handle this.